src/gs-window-x11.c

```diff
@@ -2219,57 +2250,31 @@ create_panel (GSWindow *window)
 {
 	GtkWidget    *left_hbox;
 	GtkWidget    *right_hbox;
-	GtkWidget    *alignment;
-	GtkWidget    *right_alignment;
 	GtkWidget    *image;
 	GtkSizeGroup *sg;
-	GdkRGBA       bg;
-	GdkRGBA       fg;
-	int           all_states;
 	GIcon        *gicon;
 
-	bg.red = 0;
-	bg.green = 0;
-	bg.blue = 0;
-	bg.alpha = 1.0;
-
-	fg.red = 0.8;
-	fg.green = 0.8;
-	fg.blue = 0.8;
-	fg.alpha = 1.0;
-
-	all_states = 0;
-
-	gtk_widget_override_background_color (window->priv->panel, all_states, &bg);
-	gtk_widget_override_color (window->priv->panel, all_states, &fg);
```

Comment:
Does this need to be replaced by anything, or is this just covered by GTK themes now?

Reply:
Seems to be covered by GTK themes. I saw no visual difference when applying this patch.